recipes/pygit2/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+export LIBGIT2=${PREFIX}
+
+$PYTHON setup.py install
```

Comment:
Please add the options `--single-version-externally-managed --record record.txt` after `install`.